README.md
# Ace
 	使用页面分层，模块原型的想法来构建前端页面的方式。

 	Ace_gitbook 对Ace这种方式的介绍，下载后，点击book里面点击相关的内容。 

 	shade ace 一些简单的组件的书写

 	mrda 用fires的方式来书写的一个企业站内容。 










Ace_gitbook/History.md
#History

| Date      |    修改内容 | 备注  |
| :-------- | --------:| :--: |
| 2014-10-15  | 新建文档 |  gitbook   |
| 2014-10-16  | 对前4部分内容做修改 |  gitbook不管是一个简单的文档，应该是一个可以复制和粘贴的类库  
| 2015-10-28  | 整理和完善 |     


mrda/js/slide.js

var glume = function(id){
    this.ctn = document.getElementById(id);
    this.adLis = null; //banner 里面的li元素初始化
    this.btns = null; //banner 里面的按钮元素。点击进行图片轮换
    this.animStep = 0.1;//动画速度0.1～0.9 数字越大 速度越快
    this.switchSpeed = 5;//自动播放间隔(s)
    this.defOpacity = 1; 
    this.tmpOpacity = 1;
    this.crtIndex = 0; //初始化时刻的图片内容 
    this.crtLi = null; // 初始化当前的li标签内容信息
    this.adLength = 0;// banner里面图片的长度
    this.timerAnim = null;
    this.timerSwitch = null;
    this.init();
};
//给glume加原型
glume.prototype = {
    fnAnim:function(toIndex){
        if(this.timerAnim){window.clearTimeout(this.timerAnim);}
        if(this.tmpOpacity <= 0){
            this.crtLi.style.opacity = this.tmpOpacity = this.defOpacity;
            this.crtLi.style.filter = 'Alpha(Opacity=' + this.defOpacity*100 + ')';
            this.crtLi.style.zIndex = 0;
            this.crtIndex = toIndex;
            return;
        }
        this.crtLi.style.opacity = this.tmpOpacity = this.tmpOpacity - this.animStep;
        this.crtLi.style.filter = 'Alpha(Opacity=' + this.tmpOpacity*100 + ')';
        this.timerAnim = window.setTimeout(this.fnAnim.bind(this,toIndex),50);
    },
    fnNextIndex:function(){
        return (this.crtIndex >= this.adLength-1)?0:this.crtIndex+1;
    },
    fnSwitch:function(toIndex){
        if(this.crtIndex==toIndex){return;}
        this.crtLi = this.adLis[this.crtIndex];
        for(var i=0;i<this.adLength;i++){
            this.adLis[i].style.zIndex = 0;
        }
        this.crtLi.style.zIndex = 2;
        this.adLis[toIndex].style.zIndex = 1;
        for(var i=0;i<this.adLength;i++){
            this.btns[i].className = '';
        }
        this.btns[toIndex].className = 'on'
        this.fnAnim(toIndex);
    },
    fnAutoPlay:function(){
        this.fnSwitch(this.fnNextIndex());
    },
    fnPlay:function(){
        this.timerSwitch = window.setInterval(this.fnAutoPlay.bind(this),this.switchSpeed*1000);// 图片转化动画效果
    },
    fnStopPlay:function(){
        window.clearTimeout(this.timerSwitch);
    },
    init:function(){
        this.adLis = this.ctn.getElementsByTagName('li');
        this.btns = this.ctn.getElementsByTagName('p')[0].getElementsByTagName('span');
        this.adLength = this.adLis.length;
        for(var i=0,l=this.btns.length;i<l;i++){
            with({i:i}){
                this.btns[i].index = i;
                this.btns[i].onmouseover = this.fnSwitch.bind(this,i);
            }
        }
        this.adLis[this.crtIndex].style.zIndex = 2;
        this.fnPlay();
        this.ctn.onmouseover = this.fnStopPlay.bind(this); //  当鼠标移动到图片上边是 停止图片的轮换
        this.ctn.onmouseout = this.fnPlay.bind(this);//当鼠标已开是图片继续轮换
    }
};
var player1 = new glume('mod-slide-api');

shade ace/ACE_Model_V_1.0/tab/js/tab.js

/* ========================================================
 * bootstrap-tab.js v2.3.1
 * http://twitter.github.com/bootstrap/javascript.html#tabs
 * ========================================================
 * Copyright 2012 Twitter, Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * ======================================================== */

!function ($) {
    "use strict"; // jshint ;_;


    /* TAB CLASS DEFINITION
     * ==================== */

    var Tab = function (element) {
        this.element = $(element)
    }

    Tab.prototype = {

        constructor: Tab

        , show: function () {
            var $this = this.element
                , $ul = $this.closest('ul:not(.dropdown-menu)')
                , selector = $this.attr('data-target')
                , previous
                , $target
                , e

            if (!selector) {
                selector = $this.attr('href')
                selector = selector && selector.replace(/.*(?=#[^\s]*$)/, '') //strip for ie7
            }

            if ( $this.parent('li').hasClass('active') ) return

            previous = $ul.find('.active:last a')[0]

            e = $.Event('show', {
                relatedTarget: previous
            })

            $this.trigger(e)

            if (e.isDefaultPrevented()) return

            $target = $(selector)

            this.activate($this.parent('li'), $ul)
            this.activate($target, $target.parent(), function () {
                $this.trigger({
                    type: 'shown'
                    , relatedTarget: previous
                })
            })
        }

        , activate: function ( element, container, callback) {
            var $active = container.find('> .active')
                , transition = callback
                    && $.support.transition
                    && $active.hasClass('fade')

            function next() {
                $active
                    .removeClass('active')
                    .find('> .dropdown-menu > .active')
                    .removeClass('active')

                element.addClass('active')

                if (transition) {
                    element[0].offsetWidth // reflow for transition
                    element.addClass('in')
                } else {
                    element.removeClass('fade')
                }

                if ( element.parent('.dropdown-menu') ) {
                    element.closest('li.dropdown').addClass('active')
                }

                callback && callback()
            }

            transition ?
                $active.one($.support.transition.end, next) :
                next()

            $active.removeClass('in')
        }
    }


    /* TAB PLUGIN DEFINITION
     * ===================== */

    var old = $.fn.tab

    $.fn.tab = function ( option ) {
        return this.each(function () {
            var $this = $(this)
                , data = $this.data('tab')
            if (!data) $this.data('tab', (data = new Tab(this)))
            if (typeof option == 'string') data[option]()
        })
    }

    $.fn.tab.Constructor = Tab


    /* TAB NO CONFLICT
     * =============== */

    $.fn.tab.noConflict = function () {
        $.fn.tab = old
        return this
    }


    /* TAB DATA-API
     * ============ */

    $(document).on('click.tab.data-api', '[data-toggle="tab"], [data-toggle="pill"]', function (e) {
        e.preventDefault()
        $(this).tab('show')
    })

}(window.jQuery);

mrda/images/banner/MRDA/MRDA_edgeActions.js
/***********************
* Adobe Edge Animate Composition Actions
*
* Edit this file with caution, being careful to preserve 
* function signatures and comments starting with 'Edge' to maintain the 
* ability to interact with these actions from within Adobe Edge Animate
*
***********************/
(function($, Edge, compId){
var Composition = Edge.Composition, Symbol = Edge.Symbol; // aliases for commonly used Edge classes

   //Edge symbol: 'stage'
   (function(symbolName) {
      
      
      Symbol.bindTimelineAction(compId, symbolName, "Default Timeline", "complete", function(sym, e) {
         // insert code to be run at timeline end here
         // Play the timeline at a label or specific time. For example:
         // sym.play(500); or sym.play("myLabel");
         sym.play("loop");
         

      });
      //Edge binding end

   })("stage");
   //Edge symbol end:'stage'

   //=========================================================
   
   //Edge symbol: 'note'
   (function(symbolName) {   
   
   })("note");
   //Edge symbol end:'note'

})(jQuery, AdobeEdge, "EDGE-88919355");

Ace_gitbook/_book/demo/ACE_javascript_V_1.0/ACE_Modal_V_1.5/js/modal.js

!function ($) {

  "use strict"; // jshint ;_;


  /* CSS TRANSITION SUPPORT (http://www.modernizr.com/)
   * ======================================================= */

  $(function () {

    $.support.transition = (function () {

      var transitionEnd = (function () {

        var el = document.createElement('bootstrap')
          , transEndEventNames = {
               'WebkitTransition' : 'webkitTransitionEnd'
            ,  'MozTransition'    : 'transitionend'
            ,  'OTransition'      : 'oTransitionEnd otransitionend'
            ,  'transition'       : 'transitionend'
            }
          , name

        for (name in transEndEventNames){
          if (el.style[name] !== undefined) {
            return transEndEventNames[name]
          }
        }

      }())

      return transitionEnd && {
        end: transitionEnd
      }

    })()

  })

}(window.jQuery);
/* =========================================================
 * bootstrap-modal.js v2.3.1
 * http://twitter.github.com/bootstrap/javascript.html#modals
 * =========================================================
 * Copyright 2012 Twitter, Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * ========================================================= */


/*
 * modified by @sofish
 * overview:
 *  - 窗口高度没有弹出层内容高时，把弹层上边与窗口上对齐(type: bug)
 *  - 窗口太窄，弹层左边与窗口左边对齐
 *  - CSS Hack: IE6 min-height
 */
!function( $ ){

  "use strict"

  /* MODAL CLASS DEFINITION
   * ====================== */

  var Modal = function ( content, options ) {
    this.options = options
    this.$element = $(content)
      .delegate('[data-dismiss="modal"]', 'click.dismiss.modal', $.proxy(this.hide, this))
  }

  Modal.prototype = {

    constructor: Modal

    ,
    toggle: function (data) {
      return this[!this.isShown ? 'show' : 'hide'](data)
    }
    //
    // add by @sofish
    // calculate the element height
    ,
    setPos: function(el){
      var toTop = $(document).scrollTop(),
        winHeight = $(window).height(),
        winWidth = $(window).width(),
        height = el.height(),
        width = el.width();
      el.css({
        'margin-left': -(winWidth > width ? parseInt(width/2, 10) : 0) + 'px',
        'margin-top': -(winHeight > height ? parseInt(height/2) : 0) + 'px',
        'top': (winHeight > height ? parseInt(winHeight/2) + toTop : toTop)+ 'px',
        'left': (winWidth > width ? 50 : 0) + '%'
      });
    }

    ,
    show: function (data) {
      var that = this

      if (this.isShown) return

      $('body').addClass('modal-open').append(that.$element);

      // add by @sofish
      // set margin to adjust the window
      this.setPos(that.$element);
      $(window).bind('scroll resize', function(){
        that.setPos(that.$element);
      });

      if(data) data['setPos'] = this.setPos;

      this.isShown = true
      this.$element.trigger('show', data);

      escape.call(this)
      backdrop.call(this, function () {
        var transition = $.support.transition && that.$element.hasClass('fade')

        !that.$element.parent().length && that.$element.appendTo(document.body) //don't move modals dom position

        that.$element
          .show()

        if (transition) {
          that.$element[0].offsetWidth // force reflow
        }

        that.$element.addClass('in')

        transition ?
          that.$element.one($.support.transition.end, function () {
            that.$element.trigger('shown')
          }) :
          that.$element.trigger('shown', data)

      })
    },

    hide: function ( e ) {

      e && e.preventDefault && e.preventDefault()

      if (!this.isShown) return

      var that = this
      this.isShown = false

      $('body').removeClass('modal-open')

      escape.call(this)

      this.$element
        .trigger('hide')
        .removeClass('in')

      $.support.transition && this.$element.hasClass('fade') ?
        hideWithTransition.call(this) :
        hideModal.call(this)
    }

  }


  /* MODAL PRIVATE METHODS
   * ===================== */

  function hideWithTransition() {
    var that = this
      , timeout = setTimeout(function () {
        that.$element.off($.support.transition.end)
        hideModal.call(that)
      }, 500)

    this.$element.one($.support.transition.end, function () {
      clearTimeout(timeout)
      hideModal.call(that)
    })
  }

  function hideModal( that ) {
    this.$element
      .hide()
      .trigger('hidden')

    backdrop.call(this)
  }

  function backdrop( callback ) {
    var that = this
      , animate = this.$element.hasClass('fade') ? 'fade' : ''

    if (this.isShown && this.options.backdrop) {
      var doAnimate = $.support.transition && animate

      this.$backdrop = $('<div class="modal-backdrop ' + animate + '"></div>')
        .appendTo(document.body);

      // add by @sofish
      // adjust backdrop height
      var htmlHeight = $(document).height(),
        winHeight = $(window).height(),
        backDrop = $('.modal-backdrop'),
        bdHeight = Math.max(htmlHeight, winHeight);

      backDrop.length && backDrop.height(bdHeight);

      if (this.options.backdrop != 'static') {
        this.$backdrop.click($.proxy(this.hide, this))
      }

      if (doAnimate) this.$backdrop[0].offsetWidth // force reflow

      this.$backdrop.addClass('in')

      doAnimate ?
        this.$backdrop.one($.support.transition.end, callback) :
        callback()

    } else if (!this.isShown && this.$backdrop) {
      this.$backdrop.removeClass('in')

      $.support.transition && this.$element.hasClass('fade')?
        this.$backdrop.one($.support.transition.end, $.proxy(removeBackdrop, this)) :
        removeBackdrop.call(this)

    } else if (callback) {
      callback()
    }
  }

  function removeBackdrop() {
    this.$backdrop.remove()
    this.$backdrop = null
  }

  function escape() {
    var that = this
    if (this.isShown && this.options.keyboard) {
      $(document).on('keyup.dismiss.modal', function ( e ) {
        e.which == 27 && that.hide()
      })
    } else if (!this.isShown) {
      $(document).off('keyup.dismiss.modal')
    }
  }


  /* MODAL PLUGIN DEFINITION
   * ======================= */

  $.fn.modal = function ( option, extra_data ) {
    return this.each(function () {
      var $this = $(this)
        , data = $this.data('modal')
        , options = $.extend({}, $.fn.modal.defaults, $this.data(), typeof option == 'object' && option)
      if (!data) $this.data('modal', (data = new Modal(this, options)))
      if (typeof option == 'string') data[option](extra_data)
      else if (options.show) data.show(extra_data)
    })
  }

  $.fn.modal.defaults = {
    backdrop: true
    ,
    keyboard: true
    ,
    show: true
  }

  $.fn.modal.Constructor = Modal


  /* MODAL DATA-API
   * ============== */

  $(function () {
    $('body').on('click.modal.data-api', '[data-toggle="modal"]', function ( e ) {
      var $this = $(this), href
        , $target = $($this.attr('data-target') || (href = $this.attr('href')) && href.replace(/.*(?=#[^\s]+$)/, '')) //strip for ie7
        , option = $target.data('modal') ? 'toggle' : $.extend({}, $target.data())

      e.preventDefault()
      $target.modal(option, $this.data())
    })
  })

}( window.jQuery );



mrda/images/banner/MRDA/MRDA_edge.js
/**
 * Adobe Edge: symbol definitions
 */
(function($, Edge, compId){
//images folder
var im='images/';

var fonts = {};
var opts = {
    'gAudioPreloadPreference': 'auto',

    'gVideoPreloadPreference': 'auto'
};
var resources = [
];
var symbols = {
"stage": {
    version: "4.0.0",
    minimumCompatibleVersion: "4.0.0",
    build: "4.0.0.359",
    baseState: "Base State",
    scaleToFit: "none",
    centerStage: "none",
    initialState: "Base State",
    gpuAccelerate: false,
    resizeInstances: false,
    content: {
            dom: [
            {
                id: 'pic12Copy2',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic1.png",'0%','0%','15.37%','14.75%']
            },
            {
                id: 'member',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"member2.jpg",'0%','0%','auto','95.58%']
            },
            {
                id: 'pic72',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic7.jpg",'0%','0%','15.37%','14.75%']
            },
            {
                id: 'pic62',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic6.jpg",'0%','0%','15.37%','14.75%']
            },
            {
                id: 'pic52',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic5.jpg",'0%','0%','15.37%','14.75%']
            },
            {
                id: 'pic42',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic4.jpg",'0%','0%','15.37%','14.75%']
            },
            {
                id: 'pic32',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic3.jpg",'0%','0%','15.37%','14.75%']
            },
            {
                id: 'pic22',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic2.png",'0%','0%','15.37%','14.75%']
            },
            {
                id: 'pic12',
                type: 'image',
                rect: ['0%', '0%','99.6%','95.6%','auto', 'auto'],
                fill: ["rgba(0,0,0,0)",im+"pic1.jpg",'0%','0%','15.35%','14.75%']
            },
            {
                id: 'Rectangle',
                type: 'rect',
                rect: ['1px', '1px','350px','100%','auto', 'auto'],
                fill: ["rgba(192,192,192,1)"],
                stroke: [0,"rgba(0,0,0,1)","none"],
                boxShadow: ["", 3, 3, 3, 0, "rgba(0,0,0,0.65098)"]
            },
            {
                id: 'menu',
                type: 'image',
                tag: 'img',
                rect: ['0', '0','19.8%','auto','auto', 'auto'],
                sizeRange: ['270px','270px','',''],
                fill: ["rgba(0,0,0,0)",im+"menu.png",'0px','0px','270px','180px']
            },
            {
                id: 'note',
                type: 'rect',
                rect: ['0%', 'auto','auto','auto','auto', '2.8%']
            }],
            symbolInstances: [
            {
                id: 'note',
                symbolName: 'note',
                autoPlay: {

                }
            }
            ]
        },
    states: {
        "Base State": {
            "${_pic22}": [
                ["style", "background-size", [104.20338983051,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "overflow", 'visible'],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ],
            "${_pic12}": [
                ["style", "background-size", [104.0677991964,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "height", '100%'],
                ["transform", "scaleY", '1'],
                ["style", "overflow", 'visible'],
                ["transform", "scaleX", '1'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100.11%']
            ],
            "${_pic62}": [
                ["style", "background-size", [104.20338983051,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "overflow", 'visible'],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ],
            "${_Rectangle}": [
                ["color", "background-color", 'rgba(255,255,255,1.00)'],
                ["subproperty", "boxShadow.color", 'rgba(221,221,221,1.00)'],
                ["style", "opacity", '0.50216211972198'],
                ["style", "left", '0px'],
                ["style", "width", '320px'],
                ["style", "top", '0px'],
                ["subproperty", "boxShadow.offsetV", '0px'],
                ["subproperty", "boxShadow.offsetH", '1px'],
                ["subproperty", "boxShadow.blur", '0px']
            ],
            "${_pic52}": [
                ["style", "background-size", [104.20338983051,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "overflow", 'visible'],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ],
            "${_pic72}": [
                ["style", "background-size", [104.20338983051,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "overflow", 'visible'],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ],
            "${_member}": [
                ["style", "background-size", ['auto',100], {valueTemplate:'@@0@@ @@1@@%'} ],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ],
            "${_pic12Copy2}": [
                ["transform", "scaleX", '1'],
                ["style", "background-size", [104.20338983051,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["transform", "scaleY", '1'],
                ["style", "overflow", 'visible'],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ],
            "${_note}": [
                ["style", "top", 'auto'],
                ["style", "bottom", '2.8%']
            ],
            "${_Stage}": [
                ["color", "background-color", 'rgba(255,255,255,1)'],
                ["style", "min-width", '0px'],
                ["style", "overflow", 'hidden'],
                ["style", "height", '100%'],
                ["style", "width", '100%']
            ],
            "${_menu}": [
                ["style", "top", '55px'],
                ["style", "min-width", '270.06px'],
                ["style", "max-width", '270px'],
                ["style", "height", 'auto'],
                ["style", "left", '23px'],
                ["style", "-webkit-transform-origin", [50,49], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "-moz-transform-origin", [50,49],{valueTemplate:'@@0@@% @@1@@%'}],
                ["style", "-ms-transform-origin", [50,49],{valueTemplate:'@@0@@% @@1@@%'}],
                ["style", "msTransformOrigin", [50,49],{valueTemplate:'@@0@@% @@1@@%'}],
                ["style", "-o-transform-origin", [50,49],{valueTemplate:'@@0@@% @@1@@%'}]
            ],
            "${_pic42}": [
                ["style", "background-size", [104.20338983051,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "overflow", 'visible'],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ],
            "${_pic32}": [
                ["style", "background-size", [104.20338983051,100], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "overflow", 'visible'],
                ["style", "height", '100%'],
                ["style", "opacity", '1'],
                ["style", "background-position", [50,50], {valueTemplate:'@@0@@% @@1@@%'} ],
                ["style", "width", '100%']
            ]
        }
    },
    timelines: {
        "Default Timeline": {
            fromState: "Base State",
            toState: "",
            duration: 53534,
            autoPlay: true,
            labels: {
                "loop": 0
            },
            timeline: [
                { id: "eid141", tween: [ "style", "${_pic32}", "background-position", [50,50], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,50]}], position: 0, duration: 0 },
                { id: "eid68", tween: [ "style", "${_Rectangle}", "width", '320px', { fromValue: '320px'}], position: 0, duration: 0 },
                { id: "eid145", tween: [ "style", "${_pic72}", "background-position", [50,50], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,50]}], position: 0, duration: 0 },
                { id: "eid543", tween: [ "style", "${_pic12Copy2}", "width", '100%', { fromValue: '100%'}], position: 0, duration: 0 },
                { id: "eid544", tween: [ "style", "${_pic12Copy2}", "width", '100%', { fromValue: '100%'}], position: 53534, duration: 0 },
                { id: "eid71", tween: [ "style", "${_menu}", "left", '23px', { fromValue: '23px'}], position: 0, duration: 0 },
                { id: "eid54", tween: [ "subproperty", "${_Rectangle}", "boxShadow.blur", '0px', { fromValue: '0px'}], position: 0, duration: 0 },
                { id: "eid747", tween: [ "style", "${_pic62}", "background-size", [104.20338983051,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.20338983051,100]}], position: 0, duration: 0 },
                { id: "eid748", tween: [ "style", "${_pic42}", "background-size", [104.20338983051,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.20338983051,100]}], position: 0, duration: 0 },
                { id: "eid788", tween: [ "subproperty", "${_Rectangle}", "boxShadow.color", 'rgba(221,221,221,1.00)', { fromValue: 'rgba(221,221,221,1.00)'}], position: 2000, duration: 0 },
                { id: "eid55", tween: [ "subproperty", "${_Rectangle}", "boxShadow.offsetH", '1px', { fromValue: '1px'}], position: 0, duration: 0 },
                { id: "eid742", tween: [ "style", "${_pic22}", "background-size", [104.20338983051,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.20338983051,100]}], position: 0, duration: 0 },
                { id: "eid732", tween: [ "style", "${_pic12}", "background-size", [104.0677991964,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.0677991964,100]}], position: 0, duration: 0 },
                { id: "eid143", tween: [ "style", "${_pic52}", "background-position", [50,50], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,50]}], position: 0, duration: 0 },
                { id: "eid207", tween: [ "style", "${_Rectangle}", "opacity", '0.50216211972198', { fromValue: '0.50216211972198'}], position: 0, duration: 0 },
                { id: "eid41", tween: [ "style", "${_pic12}", "opacity", '0', { fromValue: '1'}], position: 4000, duration: 2034 },
                { id: "eid38", tween: [ "style", "${_pic72}", "opacity", '0', { fromValue: '1'}], position: 44000, duration: 2034 },
                { id: "eid139", tween: [ "style", "${_pic12}", "background-position", [50,50], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,50]}], position: 0, duration: 0 },
                { id: "eid746", tween: [ "style", "${_pic72}", "background-size", [104.20338983051,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.20338983051,100]}], position: 0, duration: 0 },
                { id: "eid64", tween: [ "style", "${_Rectangle}", "top", '0px', { fromValue: '0px'}], position: 0, duration: 0 },
                { id: "eid42", tween: [ "style", "${_pic22}", "opacity", '0', { fromValue: '1'}], position: 9966, duration: 2034 },
                { id: "eid745", tween: [ "style", "${_pic32}", "background-size", [104.20338983051,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.20338983051,100]}], position: 0, duration: 0 },
                { id: "eid40", tween: [ "style", "${_pic32}", "opacity", '0', { fromValue: '1'}], position: 16000, duration: 2034 },
                { id: "eid142", tween: [ "style", "${_pic42}", "background-position", [50,50], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,50]}], position: 0, duration: 0 },
                { id: "eid36", tween: [ "style", "${_pic62}", "opacity", '0', { fromValue: '1'}], position: 36500, duration: 2034 },
                { id: "eid539", tween: [ "style", "${_pic12Copy2}", "height", '100%', { fromValue: '100%'}], position: 0, duration: 0 },
                { id: "eid74", tween: [ "style", "${_menu}", "-webkit-transform-origin", [50,49], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,49]}], position: 0, duration: 0 },
                { id: "eid875", tween: [ "style", "${_menu}", "-moz-transform-origin", [50,49], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,49]}], position: 0, duration: 0 },
                { id: "eid876", tween: [ "style", "${_menu}", "-ms-transform-origin", [50,49], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,49]}], position: 0, duration: 0 },
                { id: "eid877", tween: [ "style", "${_menu}", "msTransformOrigin", [50,49], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,49]}], position: 0, duration: 0 },
                { id: "eid878", tween: [ "style", "${_menu}", "-o-transform-origin", [50,49], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,49]}], position: 0, duration: 0 },
                { id: "eid53", tween: [ "subproperty", "${_Rectangle}", "boxShadow.offsetV", '0px', { fromValue: '0px'}], position: 0, duration: 0 },
                { id: "eid50", tween: [ "color", "${_Rectangle}", "background-color", 'rgba(255,255,255,1.00)', { animationColorSpace: 'RGB', valueTemplate: undefined, fromValue: 'rgba(255,255,255,1.00)'}], position: 0, duration: 0 },
                { id: "eid144", tween: [ "style", "${_pic62}", "background-position", [50,50], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,50]}], position: 0, duration: 0 },
                { id: "eid743", tween: [ "style", "${_pic52}", "background-size", [104.20338983051,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.20338983051,100]}], position: 0, duration: 0 },
                { id: "eid741", tween: [ "style", "${_pic12Copy2}", "background-size", [104.20338983051,100], { valueTemplate: '@@0@@% @@1@@%', fromValue: [104.20338983051,100]}], position: 0, duration: 0 },
                { id: "eid49", tween: [ "style", "${_menu}", "top", '55px', { fromValue: '55px'}], position: 0, duration: 0 },
                { id: "eid34", tween: [ "style", "${_pic52}", "opacity", '0', { fromValue: '1'}], position: 29500, duration: 2034 },
                { id: "eid32", tween: [ "style", "${_pic42}", "opacity", '0', { fromValue: '1'}], position: 22500, duration: 2034 },
                { id: "eid538", tween: [ "transform", "${_pic12Copy2}", "scaleY", '1', { fromValue: '1'}], position: 0, duration: 0 },
                { id: "eid537", tween: [ "style", "${_member}", "opacity", '0', { fromValue: '1'}], position: 51500, duration: 2034 },
                { id: "eid541", tween: [ "style", "${_pic12Copy2}", "opacity", '0', { fromValue: '1'}], position: 51500, duration: 2034 },
                { id: "eid542", tween: [ "style", "${_pic12Copy2}", "background-position", [50,50], { valueTemplate: '@@0@@% @@1@@%', fromValue: [50,50]}], position: 0, duration: 0 },
                { id: "eid63", tween: [ "style", "${_Rectangle}", "left", '0px', { fromValue: '0px'}], position: 0, duration: 0 },
                { id: "eid540", tween: [ "transform", "${_pic12Copy2}", "scaleX", '1', { fromValue: '1'}], position: 0, duration: 0 }            ]
        }
    }
},
"note": {
    version: "4.0.0",
    minimumCompatibleVersion: "4.0.0",
    build: "4.0.0.359",
    baseState: "Base State",
    scaleToFit: "none",
    centerStage: "none",
    initialState: "Base State",
    gpuAccelerate: false,
    resizeInstances: false,
    content: {
            dom: [
                {
                    type: 'rect',
                    id: 'Rectangle2',
                    stroke: [0, 'rgb(0, 0, 0)', 'none'],
                    rect: ['37px', '0px', '178.5%', '26px', 'auto', 'auto'],
                    fill: ['rgba(255,255,255,1)']
                },
                {
                    font: ['Arial, Helvetica, sans-serif', [24, ''], 'rgba(0,0,0,1)', 'normal', 'none', ''],
                    rect: ['44px', '13px', 'auto', 'auto', 'auto', 'auto'],
                    id: 'Text',
                    text: 'Website is under constuction...',
                    type: 'text',
                    tag: 'p'
                }
            ],
            symbolInstances: [
            ]
        },
    states: {
        "Base State": {
            "${_Rectangle2}": [
                ["style", "top", '0px'],
                ["style", "left", '0px'],
                ["style", "width", '100.03%']
            ],
            "${_Text}": [
                ["style", "top", '6px'],
                ["color", "color", 'rgba(153,153,153,1.00)'],
                ["style", "left", '10px'],
                ["style", "font-size", '12px']
            ],
            "${symbolSelector}": [
                ["style", "height", '26px'],
                ["style", "width", '186px']
            ]
        }
    },
    timelines: {
        "Default Timeline": {
            fromState: "Base State",
            toState: "",
            duration: 0,
            autoPlay: true,
            timeline: [
                { id: "eid226", tween: [ "style", "${_Text}", "font-size", '12px', { fromValue: '12px'}], position: 0, duration: 0 },
                { id: "eid229", tween: [ "color", "${_Text}", "color", 'rgba(153,153,153,1.00)', { animationColorSpace: 'RGB', valueTemplate: undefined, fromValue: 'rgba(153,153,153,1.00)'}], position: 0, duration: 0 },
                { id: "eid260", tween: [ "style", "${_Rectangle2}", "left", '0px', { fromValue: '0px'}], position: 0, duration: 0 },
                { id: "eid263", tween: [ "style", "${_Text}", "left", '10px', { fromValue: '10px'}], position: 0, duration: 0 },
                { id: "eid230", tween: [ "style", "${_Rectangle2}", "width", '100.03%', { fromValue: '100.03%'}], position: 0, duration: 0 },
                { id: "eid228", tween: [ "style", "${_Text}", "top", '6px', { fromValue: '6px'}], position: 0, duration: 0 }            ]
        }
    }
}
};


Edge.registerCompositionDefn(compId, symbols, fonts, resources, opts);

/**
 * Adobe Edge DOM Ready Event Handler
 */
$(window).ready(function() {
     Edge.launchComposition(compId);
});
})(jQuery, AdobeEdge, "EDGE-88919355");
